src/model.rs

```diff
@@ -95,12 +95,17 @@ impl Default for TransactionData {
 
 #[derive(Debug, Serialize, Deserialize, Default, Clone)]
 pub struct ProofTransaction {
-    pub blob_tx_hash: TxHash,
     pub contract_name: ContractName,
-    pub blob_index: BlobIndex,
+    pub blobs_references: Vec<BlobReference>,
```

Comment:
le `blob_tx_hash` reste commun à tous les blobs, donc on peut le laisser dans ProofTransaction
```suggestion
    pub blob_tx_hash: TxHash,
    pub blobs_indexes: Vec<BlobIndex>,
```